### PR TITLE
Simplyfing the evaluation step

### PR DESCRIPTION
**Pull request by yashk2810**
Simplyfing the evaluation step by taking argmax of the softmax of the predictions instead of tf.multinomial. This makes it easier to understand.

### CODE REVIEW

**Comment by yashk2810** [comment]
@alextp @yifeif

**Review by alextp — CHANGES_REQUESTED** [review]

File: tensorflow/contrib/eager/python/examples/generative_examples/image_captioning_with_attention.ipynb [hunk 1]
```diff
@@ -1056,7 +1056,7 @@
         "\n",
         "        attention_plot[i] = tf.reshape(attention_weights, (-1, )).numpy()\n",
         "\n",
-        "        predicted_id = tf.multinomial(tf.exp(predictions), num_samples=1)[0][0].numpy()\n",
+        "        predicted_id = tf.argmax(tf.nn.softmax(predictions[0])).numpy()\n",
```
[the remark this answers is not in the document]
Comment (yashk2810) [reply tensorflow/contrib/eager/python/examples/generative_examples/image_captioning_with_attention.ipynb]: Done

**Comment by yashk2810** [comment]
Done

**Review by alextp — DISMISSED** [review]

**Review by alextp — APPROVED** [review]

**Comment by yashk2810** [comment]
If all the required tests have passed, shouldn't the copybara bot merge this PR?

**Comment by alextp** [comment]
A lot of tests failed, apparently.

**Comment by yashk2810** [comment]
Is there any way I can run these tests?

**Comment by yashk2810** [comment]
@protoget Can you merge this?

**Comment by yashk2810** [comment]
This is the fifth time that these tests have failed lol 😂😂.

None of them are being caused to fail because of this PR.
